pipelines/batch/src/main/java/com/google/fhir/analytics/FhirEtl.java

```diff
@@ -514,8 +514,12 @@ static List<Pipeline> setupAndBuildPipelines(
   }
 
   public static void main(String[] args)
-      throws IOException, SQLException, ViewDefinitionException, ProfileException,
-          ExecutionException, InterruptedException {
+      throws IOException,
```

Comment:
This formatting issue seems to be because of the inconsistency in the version for IDE java-format-plugin and the spotless configuration over [here](https://github.com/google/fhir-data-pipes/blob/90fd8b7b3899acff6f1dd4ff69693cf5580866e7/pom.xml#L134) which should have been 1.17.0, do you mind changing this configuration value to 1.17.0 as well.

Reply:
Sure! Updated pom.xml